app/scenes/channel/components/channel_post_list/channel_post_list_container.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {loadPostsIfNecessary} from 'app/actions/views/channel';
+
+import {getPostsInCurrentChannel} from 'service/selectors/entities/posts';
+
+import ChannelPostList from './channel_post_list';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
+        posts: getPostsInCurrentChannel(state)
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        actions: bindActionCreators({
+            loadPostsIfNecessary
+        }, dispatch)
+    };
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(ChannelPostList);
```

Comment:
Like above, that's part of the reason for the move to redux so that they don't need to be tightly coupled like that. This way, we can just drop a post_list in anywhere with a different list of posts (like I'm thinking of the RHS) and it'll just work without anything extra needing to be hooked up